appcenter-data/ios/AppCenterReactNativeData/AppCenterReactNativeData.m

```diff
@@ -40,4 +50,24 @@ + (void)register {
     }
 }
 
+RCT_EXPORT_METHOD(read:(NSString *)documentId
+                  partition:(NSString *) partition
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject){
+    [MSData readDocumentWithID: documentId
+                  documentType:[MSDictionaryDocument class]
+                     partition: partition
+             completionHandler:^(MSDocumentWrapper *_Nonnull document) {
+                 NSMutableDictionary *dict = [[NSMutableDictionary alloc] init];
+                 dict[kMSDeserializedValue] = document.deserializedValue.serializeToDictionary;
```

Comment:
Is this pseudo code the `serializeToDictionary`?

Reply:
I'm using`serializeToDictionary` in  [MSSerializableDocument.h](https://github.com/microsoft/appcenter-sdk-apple/blob/develop/AppCenterData/AppCenterData/MSSerializableDocument.h)

Reply:
Changed to `[document.deserializedValue serializeToDictionary]`